src/main.rs

```diff
@@ -153,48 +141,58 @@ fn search_snare_conf() -> Option<PathBuf> {
 }
 
 /// If the config specified a 'user' then switch to that and update $HOME and $USER appropriately.
+/// This function *must* not be called after daemonisation.
```

Comment:
Ah. I think you should emphasize *not* instead of *must*. It's very easy to see:
> This function *must* be called after daemonisation.

Reply:
To be honest, the emphasis is overkill. Removed in https://github.com/softdevteam/snare/pull/63/commits/7ae0c1b2395c9acd4bca66adb5b56f2fd528b051.

Reply:
OK. There is at least one other instance of emphasis added in this PR.

Reply:
Removed in https://github.com/softdevteam/snare/pull/63/commits/4faf5eea066d3647b5b034509310d66250078eae.